### PR TITLE
Update runtime version

### PR DESCRIPTION
**Pull request by ryanbrandenburg**
Tested this locally against Universe command line and logging in VS.

### CODE REVIEW

**Comment by ryanbrandenburg** [comment]
It appears that this won't work on a clean machine because it can't find ['Microsoft.NETCore.App', version '2.0.0'](https://travis-ci.org/aspnet/BuildTools/builds/315103143#L714).

File: files/KoreBuild/config/runtime.version [hunk 1]
```diff
@@ -1 +1 @@
-2.0.3
```
Comment (natemcmaster) [inline files/KoreBuild/config/runtime.version]: You should only change this if you also re-target all build tools to 2.1.

The repos themselves take care of installing the runtime via the `<DotNetRuntime>` item group. example:  https://github.com/aspnet/Mvc/blob/dev/build/repo.props#L15-L18

**Comment by natemcmaster** [comment]
IMO, it would be better not to take this yet until .NET Core 2.1 is stable. Historically, we've run in roll-forward issues when our build tools depend on pre-release runtimes.

**Comment by JunTaoLuo** [comment]
Spoke to @natemcmaster, yea it looks like we don't want to make the changes here. The version of the shared framework used by the universe build is actually controlled via https://github.com/aspnet/Universe/blob/dev/build/dependencies.props#L46

**Comment by ryanbrandenburg** [comment]
@JunTaoLuo is the value in Universe high enough for your purposes?

Also, closing.

**Comment by JunTaoLuo** [comment]
@ryanbrandenburg I'll test it and yell really loud if it isn't.

**Pull request closed without merging**